deviceplugin/usb.go

```diff
@@ -241,43 +241,49 @@ func searchUSBDevices(devices *[]usbDevice, vendor USBID, product USBID) (devs [
 func (gp *GenericPlugin) discoverUSB() (devices []device, err error) {
 	for _, group := range gp.ds.Groups {
 		var paths []string
-		usbDevs, err := enumerateUSBDevices(usbDevicesDir)
-		if err != nil {
-			return devices, err
-		}
-		for _, dev := range group.USBSpecs {
-			matches, err := searchUSBDevices(&usbDevs, dev.Vendor, dev.Product)
+
+		if !gp.skipUSBEnumeration {
```

Comment:
let's invert this so that we avoid all of the extra nesting:
```suggestion
		if gp.skipUSBEnumeration {
			return
		}
```

Reply:
Ahh i love that second idea. Let's go with that: only enable USB device discovery if a USB device spec is defined